airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -203,7 +203,7 @@ class TaskInstance(StrictBaseModel):
     dag_id: str
     run_id: str
     try_number: int
-    map_index: int = -1
+    map_index: int | None = -1
```

Comment:
```suggestion
    map_index: int = -1
```

We can revert this annotation as well?